src/Features/Core/Portable/FeaturesResources.Designer.cs

```diff
@@ -436,18 +436,16 @@ internal class FeaturesResources {
                 return ResourceManager.GetString("Adding_an_imported_method_will_prevent_the_debug_session_from_continuing", resourceCulture);
             }
         }
-
+        
         /// <summary>
         ///   Looks up a localized string similar to Alias ambiguous type &apos;{0}&apos;.
         /// </summary>
-        internal static string Alias_ambiguous_type_0
-        {
-            get
-            {
+        internal static string Alias_ambiguous_type_0 {
```

Comment:
This file could be reverted

Reply:
What is checked in is wrong (maybe someone manually edited it?).  until it is fixed, anyone who edits these resx will have this change.